aws-ssm-patchbaseline/aws-ssm-patchbaseline.json

```diff
@@ -280,14 +280,19 @@
         "Name"
     ],
     "createOnlyProperties": [
-        "/properties/OperatingSystem"
+        "/properties/OperatingSystem",
+        "/properties/DefaultBaseline",
```

Comment:
why do you add tag/defaultBaseline to createOnly, 
my understanding is that customer should be able to change them.